### ISSUE TITLE
Object List items should have a button or link role for screen readers

### ISSUE THREAD

**Issue by emclaren** [mozilla/hubs#2202]
**Description**
The items in the object list should read as a button or link role type because they have the functionality that they are interactive. 

![image(4)](https://user-images.githubusercontent.com/6835324/76281502-7bad0b80-6252-11ea-9fc4-c7d803177781.png)

**Additional context**
This was flagged by the a11y community using Hubs
